lib/browser/api/module-list.js

```diff
@@ -0,0 +1,23 @@
+// Browser side modules, please sort alphabetically.
+module.exports = [
+  {name: 'app', file: 'app'},
```

Comment:
> This might be overkill here, but I figured I'd mention it.

Yeah, I'd prefer not to pull in any modules, Electron does not current ship any 3rd party modules.

> You could also add a sort call to ensure that the list is alphabetical.

The sorting is for the reader, just to make it easier to scan the list.